docs/Xamarin.Forms/ContentPage.xml

```diff
@@ -55,7 +55,7 @@ namespace ContentPageExample
     }
 }
           ]]></code>
-        <para>The FormsGallery sample, which can be found on the <format type="text/html"><a href="https://developer.xamarin.com/samples/tag/Xamarin.Forms/" target="_blank">Sample Applications</a></format> page, has a ContentPageDemoPage.cs file. This file contains a longer and more complicated example.</para>
+        <para>The <format type="text/html"><a href="https://docs.microsoft.com/en-us/samples/xamarin/xamarin-forms-samples/formsgallery/" target="_blank">FormsGallery</a></format> sample, which can be found on the <format type="text/html"><a href="https://docs.microsoft.com/en-us/samples/browse/" target="_blank">Sample Applications</a></format> page, has a ContentPageDemoPage.cs file. This file contains a longer and more complicated example.</para>
```

Comment:
We can safely remove locale from these URL's so that if the user is browsing in a different locale they get the right version (if it exists, if not it will default to this anyway).
```suggestion
        <para>The <format type="text/html"><a href="https://docs.microsoft.com/samples/xamarin/xamarin-forms-samples/formsgallery/" target="_blank">FormsGallery</a></format> sample, which can be found on the <format type="text/html"><a href="https://docs.microsoft.com/samples/browse/" target="_blank">Sample Applications</a></format> page, has a ContentPageDemoPage.cs file. This file contains a longer and more complicated example.</para>
```

Reply:
oh, cool! the more you know :)